src/Altinn.Notifications/Mappers/OrderMapper.cs

```diff
@@ -16,7 +16,7 @@ public static class OrderMapper
     /// <summary>
     /// Maps a <see cref="EmailNotificationOrderRequestExt"/> to a <see cref="NotificationOrderRequest"/>
     /// </summary>
-    public static NotificationOrderRequest MapToOrderRequest(this EmailNotificationOrderRequestExt extRequest, string creator)
+    public static NotificationOrderRequest ForEmailMapToOrderRequest(this EmailNotificationOrderRequestExt extRequest, string creator)
```

Comment:
A bit unsure why we need to rename this ForEmail, the type it maps from/to is given based on the type of the first input parameter.